packages/editor-ui/src/views/SetupWorkflowFromTemplateView/SetupWorkflowFromTemplateView.vue

```diff
@@ -0,0 +1,199 @@
+							<n8n-loading class="appCredential appCredentialBorder" variant="p" :rows="3" />
+						</div>
+					</div>
+
+					<div class="actions">
+						<n8n-link :href="skipSetupUrl" :newWindow="false" @click="onSkipSetup($event)">{{
+							$locale.baseText('templateSetup.skip')
+						}}</n8n-link>
+
+						<n8n-tooltip
+							v-if="isReady"
+							:content="buttonTooltip"
+							:disabled="setupTemplateStore.numCredentialsLeft === 0"
+						>
+							<n8n-button
+								:label="$locale.baseText('templateSetup.continue.button')"
+								:disabled="setupTemplateStore.numCredentialsLeft > 0 || setupTemplateStore.isSaving"
+								@click="setupTemplateStore.createWorkflow($router)"
+							/>
+						</n8n-tooltip>
+						<div v-else>
+							<n8n-loading variant="button" />
+						</div>
+					</div>
+				</div>
+			</div>
+		</template>
+	</TemplatesView>
+</template>
+
+<script lang="ts">
+import { mapStores } from 'pinia';
+import { defineComponent } from 'vue';
+import N8nHeading from 'n8n-design-system/components/N8nHeading';
+import N8nLink from 'n8n-design-system/components/N8nLink';
+import { useSetupTemplateStore } from './setupTemplate.store';
+import AppsRequiringCredsNotice from './AppsRequiringCredsNotice.vue';
+import SetupTemplateFormStep from './SetupTemplateFormStep.vue';
+import { externalHooks } from '@/mixins/externalHooks';
+import { VIEWS } from '@/constants';
+import TemplatesView from '../TemplatesView.vue';
+
+export default defineComponent({
+	name: 'SetupWorkflowFromTemplateView',
+	mixins: [externalHooks],
+	components: {
+		TemplatesView,
+		N8nHeading,
+		N8nLink,
+		AppsRequiringCredsNotice,
+		SetupTemplateFormStep,
+	},
+	computed: {
+		...mapStores(useSetupTemplateStore),
+		templateId() {
+			return Array.isArray(this.$route.params.id)
+				? this.$route.params.id[0]
+				: this.$route.params.id;
+		},
+		title() {
+			return this.setupTemplateStore.template?.name ?? 'unknown';
+		},
+		isReady() {
+			return !this.setupTemplateStore.isLoading;
+		},
+		skipSetupUrl() {
+			const route = this.$router.resolve({
+				name: VIEWS.TEMPLATE_IMPORT,
+				params: { id: this.templateId },
+			});
+
+			return route.fullPath;
+		},
+		buttonTooltip() {
+			const numLeft = this.setupTemplateStore.numCredentialsLeft;
+
+			return this.$locale.baseText('templateSetup.continue.tooltip', {
+				adjustToNumber: numLeft,
+				interpolate: { numLeft: numLeft.toString() },
+			});
+		},
+	},
+	watch: {
+		templateId(newTemplateId) {
+			this.setupTemplateStore.setTemplateId(newTemplateId);
+			void this.setupTemplateStore.loadTemplateIfNeeded();
+		},
+	},
+	methods: {
+		async onSkipSetup(event: MouseEvent) {
+			event.preventDefault();
+
+			await this.setupTemplateStore.skipSetup({
+				$externalHooks: this.$externalHooks(),
+				$telemetry: this.$telemetry,
+				$router: this.$router,
+			});
+		},
+		async skipIfTemplateHasNoCreds() {
```

Comment:
Nitpick: Can this be moved to the previous step (template page) so we are not event go to this page if there is not need?

Reply:
That's a valid point. I'll address this in a separate PR